dev-libs/elftoolchain/elftoolchain-0.7.1.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Libraries/utilities to handle ELF objects (BSD drop in replacement for libelf)"
+HOMEPAGE="https://wiki.freebsd.org/LibElf"
+SRC_URI="https://netcologne.dl.sourceforge.net/project/elftoolchain/Sources/${P}/${P}.tar.bz2"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	app-arch/libarchive:*
```

Comment:
Wait, really? (I may need to read up on the previous discussion)

Reply:
Actually libelf doesn't need libarchive, but I see it's needed for other toolchain binaries in elftoolchain, but since we're not using anything else I think we can skip building other stuff and drop libarchive completely. I'll look into it further

Reply:
I've removed libarchive as we don't need it at all for libelf, other tools from the source use it